euphony/src/main/java/co/euphony/ui/viewmodel/EuphonyRxPanelViewModel.kt

```diff
@@ -0,0 +1,66 @@
+package co.euphony.ui.viewmodel
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import co.euphony.rx.AcousticSensor
+import co.euphony.rx.EuRxManager
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.launch
+
+class EuphonyRxPanelViewModel(
+    private val rxManager: EuRxManager
+) : ViewModel() {
+
+    private val _isListening = MutableStateFlow(false)
+    val isListening: StateFlow<Boolean> = _isListening
+
+    private val _isListenStarted = MutableStateFlow(false)
+    val isListenStarted: StateFlow<Boolean> = _isListenStarted
```

Comment:
I think `isListenStarted` is not needed, so we can eliminate this variable :smile:

Reply:
I think `isListenStarted` is needed to define textBackgroundColor in EuphonyRxPanel.

I intended to maintain background color to `LightSkyBlue` if user don't click `listen button` even once.

But, after user click the button more than once, background color must be one of 'LightGreen' or 'LightRed'.

Am I missing something? or could you recommend me better way?

Reply:
@haan823 Oh, I now checked your code where `isListenStarted` is used! :pray: 
Now, I understand why you declared `isListenStarted`.
Another way can be utilizing `limitTime`.
If `limitTime is <= 0`, it has result, so we can write code like below:
```kotlin
    val textBackgroundColor =
        if (!isListening && viewModel.getTime() <= 0) {
            if (viewModel.isSuccess()) {
                LightGreen
            } else {
                LightRed
            }
        } else {
            LightSkyBlue
        }
```

Reply:
Oh, I think it is better code! Thank you 😄